src/sardana/pool/poolcontroller.py

```diff
@@ -84,6 +84,8 @@ def get_ctrl_error_str(self):
         err = self._ctrl_error
         if err is None:
             return ""
+        if type(err) == str:
```

Comment:
You should probably use `isinstance(err, str)` here, it's recommended by PEP8 and supposedly also faster :)